Unique_Numbers/unique_number.cpp

```diff
@@ -0,0 +1,40 @@
+#include<iostream>
+#include<stdlib.h>
+#include<algorithm>
+using namespace std;
```

Comment:
Remove this and move your code to this directory: https://github.com/OpenGenus/cosmos/tree/master/code/unclassified/src/unique_number